# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../gscmxx/compile-fix.patch
# Copyright (C) 2006 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

diff -Nur gscmxx-0.4.1-orig/Makefile.PL gscmxx-0.4.1/Makefile.PL
--- gscmxx-0.4.1-orig/Makefile.PL	2002-09-01 12:23:14.000000000 +0000
+++ gscmxx-0.4.1/Makefile.PL	2006-01-26 11:50:52.004011500 +0000
@@ -197,12 +197,12 @@
 sub postamble {
 	my $postamble = <<'END';
 pixmaps ::
-	$(MKPATH) $(PREFIX)/share/pixmaps/gscmxx
-	@$(MOD_INSTALL) ./pixmaps/ $(PREFIX)/share/pixmaps/gscmxx/
+	$(MKPATH) $(DESTDIR)$(VENDORPREFIX)/share/pixmaps/gscmxx
+	@$(MOD_INSTALL) ./pixmaps/ $(DESTDIR)$(VENDORPREFIX)/share/pixmaps/gscmxx/
 
 i18n ::
-	$(MKPATH) $(PREFIX)/share/gscmxx/i18n
-	@$(MOD_INSTALL) ./share/i18n/ $(PREFIX)/share/gscmxx/i18n/
+	$(MKPATH) $(DESTDIR)$(VENDORPREFIX)/share/gscmxx/i18n
+	@$(MOD_INSTALL) ./share/i18n/ $(DESTDIR)$(VENDORPREFIX)/share/gscmxx/i18n/
 
 END
 	return $postamble;
